core-scala/src/test/scala/com/baeldung/scala/GenericsIntro.scala

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.scala
+
+object GenericsIntro {
+  case class Apple(name: String)
```

Comment:
Is this ever used?

Reply:
Not in the code. It was mentioned in the article inside a sentence. So we should only have code that is used within the scala file(s), right? Thanks.

Reply:
I don't think this adds much, so let's lose it.

Reply:
I've changed my mind... Let's keep it, but let's provide the code that would crash in the AnyRef context. More about this in my review of the article, which is on its way.